src/shared_realm.cpp

```diff
@@ -891,6 +891,12 @@ void Realm::close()
     m_coordinator = nullptr;
 }
 
+void Realm::shutdown_sync()
+{
+    if (m_coordinator)
+        m_coordinator->shutdown_sync();
+}
```

Comment:
I'll change this to error if `m_coordinator` is null in order to catch cases where close incorrectly comes first

Reply:
Changed